lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 72px;
+$nx-sidebar-logo-height: 34px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  color: $nx-sidebar-text-color;
+  font-size: $nx-font-size;
+  max-width: $nx-sidebar-max-width;
+  padding: $nx-spacing-l 0;
+  width: $nx-sidebar-width__open;
+
+  .nx-text-link {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-page-sidebar-operable__header {
+  @include container-horizontal;
+
+  display: flex;
+  padding: $nx-spacing-sm $nx-spacing-l $nx-spacing-xl $nx-spacing-l;
+}
+
+.nx-page-sidebar-operable__toggle {
+  border-color: transparent;
+  border-radius: 50%;
+  color: $nx-sidebar-text-color;
+  flex: none;
+  height: 40px;
+  margin: 0;
+  width: 40px;
+
+  &:focus {
+    border: $nx-sidebar-focus-border;
+  }
+
+  &:active {
+    background-color: $nx-sidebar-active-color;
+  }
+
+  &:hover {
+    background-color: $nx-sidebar-hover-color;
+  }
+}
+
+.nx-page-sidebar-operable__product-info {
+  @include container-horizontal;
+
+  align-items: center;
+  color: $nx-sidebar-text-color;
+  display: flex;
+  flex: 1 1 auto;
+  text-decoration: none;
+}
+
+.nx-page-sidebar-operable__logo {
+  height: $nx-sidebar-logo-height;
+  margin-right: $nx-spacing-xs;
+}
+
+.nx-page-sidebar-operable__content {
+  color: $nx-sidebar-text-color;
+  margin: $nx-spacing-l $nx-spacing-md;
+}
+
+.nx-sidebar-navigation {
+  border-bottom: 1px solid $nx-sidebar-separator-color;
+  flex: none;
+  margin-bottom: $nx-spacing-md;
+  padding-bottom: $nx-spacing-xxl;
+}
+
+.nx-sidebar-navigation__link {
+  @include container-horizontal;
+
+  align-items: center;
+  border-color: transparent;
+  display: flex;
+  padding: 0 $nx-spacing-l;
+
+  .nx-icon {
+    margin: $nx-spacing-xs $nx-spacing-xs $nx-spacing-xs 0;
```

Comment:
Rather than the vertical margins here, I think it would be more consistent with how we've implemented other things (such as `.nx-btn`) if you add 4px vertical padding to the `<a>`.

Reply:
> Rather than the vertical margins here, I think it would be more consistent with how we've implemented other things (such as `.nx-btn`) if you add 4px vertical padding to the `<a>`.

Why is that desirable? These margins work in both closed and open states. If I do what you suggest I'd need to craft margins for the closed state to look right, leading to _more_ code.

Reply:
> If I do what you suggest I'd need to craft margins for the closed state to look right, leading to more code.

Ah I didn't consider the closed state.